packages/lib/integration/service.ts

```diff
@@ -3,13 +3,18 @@ import "server-only";
 import { prisma } from "@formbricks/database";
 import { Prisma } from "@prisma/client";
 import { DatabaseError } from "@formbricks/types/v1/errors";
-import { TIntegration } from "@formbricks/types/v1/integrations";
+import { ZId } from "@formbricks/types/v1/environment";
+import { TIntegration, TGoogleSheetIntegrationUpsert } from "@formbricks/types/v1/integrations";
 import { cache } from "react";
+import { validateInputs } from "../utils/validate";
+import { ZString } from "@formbricks/types/v1/common";
 
 export async function createOrUpdateIntegration(
   environmentId: string,
-  integrationData: any
+  integrationData: TGoogleSheetIntegrationUpsert
```

Comment:
Not specific to GoogleSheet integration but could any kind of Integration (e.g. Slack Integration currently in development)

Reply:
Done